Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿class NumberManpulator
+{
+    static double calc(double a,double b)
+    {
+        return (Math.Sin(a)+Math.Sqrt(a))/(Math.Cos(b)+Math.Sqrt(b));
+
+    }
+    static void Main(string[] args)
+    {
+        double x = NumberManpulator.calc(3,5)*NumberManpulator.calc(3,7)*NumberManpulator.calc(5,5);
+        Console.WriteLine(Math.Round(x,3));
+
+    }

```

Comment:
Постарайтесь впредь придерживаться следующих рекомендаций:

1. Методы в классе старайтесь располагать в порядке вызовов. Сначала вызывающие, потом вызываемые. Так код читается лучше.
2. Методы в классе разделяйте пустой строкой. Так код читается лучше.
3. Пустая строка перед закрывающей фигурной скобкой затрудняет чтение кода. Она не нужна.
4. Отделяйте знаки операций пробелами. Также ставьте пробелы после запятых. Так код читается лучше.
5. Старайтесь не использовать в методах лишние параметры, например лучше `Main()`, чем `Main(string[] args)`.
6. Называтйе  методы с большой буквы. В C# принято использовать имена методов в стиле PascalCase.
7. Если метод вызывается в классе, в котором он определен, имя класса перед именем метода можно опускать. Пользуйтесь этим. Так код читается лучше.

```suggestion
    static void Main()
    {
        double x = Сalc(3, 5) * Сalc(3, 7) * Сalc(5, 5);
        Console.WriteLine(Math.Round(x, 3));
    }

    static double Сalc(double a,double b)
    {
        return (Math.Sin(a) + Math.Sqrt(a)) / (Math.Cos(b) + Math.Sqrt(b));
    }
```